src/universal/components/modals/GlobalMenuModalContainer/GlobalMenuModalContainer.mobile.tsx

```diff
@@ -0,0 +1,51 @@
+import { compose } from 'redux';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import styled, { StyledFunction } from 'styled-components';
+import { withRouter } from 'react-router';
+import { connect } from 'react-redux'; 
+import DefaultModalContainer from '@components/modals/DefaulModalContainer/DefaultModalContainer.mobile';
+import GlobalMenuModal from '@components/modals/GlobalMenuModal/GlobalMenuModal.mobile';
+
+class GlobalMenuModalContainer extends React.Component<GlobalMenuModalContainerProps, any> {
+  constructor(props) {
+    super(props);
+    this.handleClickSignIn = this.handleClickSignIn.bind(this);
+  }
+
+  handleClickSignIn(e) {
+    this.props.history.push('/signin');
+    this.props.handleClickBackdrop();
+  }
+
+  render() {
+    return (
+      <DefaultModalContainer
+        handleClickBackdrop={this.props.handleClickBackdrop}>
+        <GlobalMenuModal
+          handleClickBackdrop={this.props.handleClickBackdrop}
+          handleClickSignIn={this.handleClickSignIn}/>
+    </DefaultModalContainer>
+    );
+  }
+}
+
+interface GlobalMenuModalContainerProps {
```

Comment:
I would suggest you to define interface at the very bottom of file

order as following
```
export js statement
js statement
export interface
interface
```